PCbuild/tcltk.props

```diff
@@ -17,15 +17,20 @@
     <tcltkDir Condition="$(tcltkDir) == ''">$(ExternalsDir)tcltk-$(TclVersion)\$(ArchName)\</tcltkDir>
     <tclWin32Exe Condition="$(Platform) == 'Win32'">$(tcltkDir)\bin\tclsh$(TclMajorVersion)$(TclMinorVersion)t.exe</tclWin32Exe>
     <tclWin32Exe Condition="$(Platform) != 'Win32'">$(tcltkDir)\..\win32\bin\tclsh$(TclMajorVersion)$(TclMinorVersion)t.exe</tclWin32Exe>
+    <tclExternalTommath Condition="$(TclMajorVersion) == '9'">TCL_WITH_EXTERNAL_TOMMATH;</tclExternalTommath>
```

Comment:
This is not used anywhere, just remove it.

Reply:
Actually, the TCL_WITH_EXTERNAL_TOMMATH constant **is** used when building against TclTk 9.0; although it is not used by 8.6.  With 9.0, if it is not defined then all of the symbols defined in libtommath.dll are reported by the linker as being undefined.  I believe that this is related to how the Tcl stubs library works;  it is not a straightforward linker error since libtommath.dll is being passed to the linker.

One might hope that the existence of a constant named TCL_WITH_EXTERNAL_TOMMATH indicates that one could choose not to have an external libtommath.dlll and instead arrange for libtommath to be statically linked into libtcl.dll, as is the case with Tcl 8.6.  However, I was not able to find a combination of build options which accomplished that.  I will try to find out if it is possible.

Reply:
@zooba: As far as I can tell. the following command uses MSVC to build a static library tcl90s.lib which has both zlib and (part of) libtommath built-in:
 `nmake -f makefile.vc TCL_WITH_INTERNAL_ZLIB=1 TCL_WITH_EXTERNAL_TOMMATH=0 STATIC_BUILD=1`

According to dumpbin, the tclsh90s.exe executable built by that command does not have libtommath.dll or zlib.dll as dependencies.  So presumably the same would be true of the _tkinter extension if it were linked against tcl90s.lib. 

If you are willing to use gcc in msys64 then `configure --disable-static` seems to do the equivalent thing